client/app/license/spla/add/license-spla-add.html

```diff
@@ -1,123 +1,118 @@
-                    <label for="availableServers"
-                           class="control-label"
-                           data-translate="license_spla_add_step1_server">
-                    </label>
-                    <select class="form-control"
-                            id="availableServers"
-                            name="availableServers"
-                            data-ng-options="type for type in model.options.availableServers"
-                            data-ng-model="model.selected.server">
-                        <option selected="selected"
-                                value=""
-                                data-ng-disabled="true"
-                                data-translate="license_spla_add_step1_server_placeholder">
-                        </option>
-                    </select>
-                </div>
-            </div>
+<div data-wizard
+     data-wizard-on-cancel="$ctrl.close.bind($ctrl)"
+     data-wizard-on-finish="$ctrl.addSpla.bind($ctrl)"
+     data-wizard-title=":: 'license_spla_add_title' | translate"
+     data-wizard-bread-crumb>
+
+    <div data-wizard-step
+         data-wizard-step-valid="$ctrl.selected.server"
+         data-wizard-step-on-load="$ctrl.load.bind($ctrl)">
+
+        <div class="text-center"
+             data-ng-show="!$ctrl.options.availableServers">
+            <oui-spinner></oui-spinner>
+        </div>
 
-            <p class="text-danger"
-               data-ng-show="model.options.availableServers.length != null && !model.options.availableServers.length"
-               data-translate="license_spla_add_step1_server_unavailable">
+        <div data-ng-show="$ctrl.options.availableServers.length">
+            <p class="font-weight-bold"
+               data-translate="license_spla_add_step1_question">
             </p>
-
+            <div class="form-group">
+                <label for="availableServers"
+                       class="control-label"
+                       data-translate="license_spla_add_step1_server">
+                </label>
+                <select class="form-control"
+                        id="availableServers"
+                        name="availableServers"
+                        data-ng-options="type for type in $ctrl.options.availableServers"
+                        data-ng-model="$ctrl.selected.server">
+                    <option selected="selected"
+                            value=""
+                            data-ng-disabled="true"
+                            data-translate="license_spla_add_step1_server_placeholder">
+                    </option>
+                </select>
+            </div>
         </div>
 
-        <div data-wizard-step
-             data-wizard-step-on-load="loadTypes"
-             data-wizard-step-valid="model.selected.type && model.selected.serial">
+        <p class="text-danger"
+           data-ng-show="$ctrl.options.availableServers.length != null && !$ctrl.options.availableServers.length"
```

Comment:
You should use ng-if instead of ng-show

I don't think this paragraph is displayed / not displayed / displayed / not displayed and it's specific imo. Same for the previous ng-show :)